server/server.go

```diff
@@ -57,6 +57,8 @@ type Server struct {
 	CannedPath   string        `short:"c" long:"canned-path" description:"Path to directory of pre-canned application layouts (/usr/share/chronograf/canned)" env:"CANNED_PATH" default:"canned"`
 	TokenSecret  string        `short:"t" long:"token-secret" description:"Secret to sign tokens" env:"TOKEN_SECRET"`
 	AuthDuration time.Duration `long:"auth-duration" default:"720h" description:"Total duration of cookie life for authentication (in hours). 0 means authentication expires on browser close." env:"AUTH_DURATION"`
+	// TODO(desa): think of a better name
+	NewUsersNotSuperAdmin bool `long:"new-users-not-super-admin" description:"All new users will not be given the Super Admin privilege" env:"NEW_USRES_NOT_SUPER_ADMIN"`
```

Comment:
1. typo at `NEW_USRES` --> `NEW_USERS`
1. make SUPER_ADMIN --> SUPERADMIN, thus: `NEW_USERS_NOT_SUPERADMIN`
1. make super-admin --> superadmin, thus: `new-users-not-superadmin`
1. replace the word `privilege` with `status`, per previous discussion